docs/trace/getting-started-aspnetcore/Program.cs

```diff
@@ -27,8 +27,7 @@
         .AddService(serviceName: "OTel.NET Getting Started"))
     .WithTracing(builder => builder
         .AddAspNetCoreInstrumentation()
-        .AddConsoleExporter())
-    .StartWithHost();
+        .AddConsoleExporter());
```

Comment:
I am not able to recollect the behavior.. could you help clarify the behavior when something fails ? Does this throw and allow fail-fast? Or we swallow exceptions and log, and proceed?

Reply:
As of https://github.com/open-telemetry/opentelemetry-dotnet/pull/4006 we will throw from the `IHostedService` `StartAsync` method if something throws during "build" phase.